data/sql/updates/pending_db_world/rev_1623976434519989200.sql

```diff
@@ -0,0 +1,56 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1623976434519989200');
+
+-- acore_string_locale
+DROP TABLE IF EXISTS `acore_string_locale`;
+CREATE TABLE IF NOT EXISTS `acore_string_locale` (
+  `entry` int(10) unsigned NOT NULL DEFAULT '0',
+  `locale` varchar(4) NOT NULL,
+  `content` text,
```

Comment:
Please read https://www.azerothcore.org/wiki/sql-standards

Reply:
> Please read https://www.azerothcore.org/wiki/sql-standards

Ok, I will read it, but it would be good, if you could quote me an example please, because in the documentation there is none about creating a table. The only thing I did, was to take the examples found in other PR.

Reply:
There is a Header about Tables and Columns... https://www.azerothcore.org/wiki/sql-standards#tables-and-columns

Reply:
I already made a commit with the fix, what is the error?

Reply:
It is good now.

Reply:
Thank you.